core/templates/dev/head/components/summary_tile/ExplorationSummaryTileDirective.js

```diff
@@ -78,11 +78,11 @@ oppia.directive('explorationSummaryTile', [
         );
       },
       controller: [
-        '$scope', '$http',
+        '$scope', '$http', '$rootScope',
```

Comment:
Don't use $rootScope. That is effectively a global namespace. You'll need to refactor so that the variables can be stored locally within their corresponding directives and controllers.

Reply:
Done